imei.sh

```
@@ -422,6 +432,11 @@ if [ -z "$JXL_VER" ]; then
   JXL_HASH=$(httpGet "https://raw.githubusercontent.com/SoftCreatR/imei/main/versions/libjxl.hash")
 fi
 
+if [ -z "$SVT_VER" ]; then
+  SVT_VER=$(httpGet "https://raw.githubusercontent.com/SoftCreatR/imei/main/versions/svt.version")
+  SVT_HASH=$(httpGet "https://raw.githubusercontent.com/SoftCreatR/imei/main/versions/svt.hash")
```

Shellcheck

[warning] SVT_HASH appears unused. Verify use (or export if used externally).

(SC2034)